configfiles/config_LctopK0s_data_tree.yml

```diff
@@ -0,0 +1,17 @@
+isMC: no
+filename: ~/DmesonAnalysis/AnalysisResults/13TeV/data/AnalysisResults_LcpK0s_2016_kl.root
```

Comment:
Is it intended to use only one period instead of the full sample (namely 2016, 2017, and 2018)?

Reply:
No, but this file is not used in the analysis yet. Anyways I can remove it you prefer, otherwise, I will upload the updated version when it's time.